pulp.spec

```diff
@@ -476,6 +476,8 @@ KEY_PATH="$KEY_DIR/rsa.key"
 KEY_PATH_PUB="$KEY_DIR/rsa_pub.key"
 if [ ! -f $KEY_PATH ]
 then
+  # Ensure the key generated is only readable by the owner.
+  umask 077
```

Comment:
The original umask doesn't get set back. Could this introduce unintended consequences, or would it be good to stash the current umask and reset it after generating this key?